src/main/java/com/wavefront/jersey/sdk/MetricNameUtils.java

```diff
@@ -0,0 +1,86 @@
+package com.wavefront.jersey.sdk;
```

Comment:
package structure as com.wavefront.sdk.jersey similar to the wavefront-java-sdk?

Reply:
As long as we are consistent